Source/WebCore/page/ChromeClient.h

```diff
@@ -668,9 +668,13 @@ class ChromeClient {
 
     virtual void proofreadingSessionUpdateStateForSuggestionWithID(const WritingTools::SessionID&, WritingTools::TextSuggestionState, const WritingTools::TextSuggestionID&) { }
 
-    virtual void removeTextAnimationForID(const WritingTools::SessionID&) { }
+    virtual void removeTextAnimationForAnimationID(const WritingTools::SessionID&) { }
 
-    virtual void cleanUpTextAnimationsForSessionID(const WritingTools::SessionID&) { }
+    virtual void removeTransparentMarkersForSessionID(const WritingTools::SessionID&) { }
+
+    virtual void removeInitialTextAnimation(const WritingTools::SessionID&) { }
+
+    virtual void addInitialTextAnimation(const WritingTools::SessionID&) { }
```

Comment:
maybe these should be named
```
removeTextAnimationForAnimationWithID
removeTransparentMarkersForSessionWithID
removeInitialTextAnimationForSessionWithID
addInitialTextAnimationForSessionWithID
```

also, for the first one, it probably shouldn't take a `SessionID` type

Reply:
I don't know that I agree with the 'with', seems extra wordy for no reason.

Reply:
ok; I think the latter two methods should at least then have `ForSessionID` added to them to avoid confusion (and the parameter type of the first one should not be SessionID)